packages/use-contractkit/src/screens/metamask.tsx

```diff
@@ -3,14 +3,14 @@ import Loader from 'react-loader-spinner';
 
 import { AddCeloNetworkButton } from '../components/AddCeloNetworkButton';
 import { UnsupportedChainIdError } from '../connectors';
-import { useMetaMaskConnector } from '../connectors/useMetaMaskConnector';
+import { useInjectedConnector } from '../connectors/useMetaMaskConnector';
 import { ConnectorProps } from '.';
 
-export const MetaMaskWallet: React.FC<ConnectorProps> = ({
+export const MetamaskOrInjectedWallet: React.FC<ConnectorProps> = ({
```

Comment:
Nitpick here, let's stay consistent with the name as MetaMask instead of Metamask